src/components/Editor/Utilities/ColorParser.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <div><span v-html="parsedText" /> <code v-if="showRaw">({{ text }})</code></div>
+</template>
+
+<script>
+export default {
+  props: ['text', 'showRaw'],
+  created() {
+    const randomize = this.randomize;
+    setInterval(() => {
+      Array.from(document.getElementsByClassName('mc-obfuscated')).forEach((element) => {
+        element.textContent = randomize(element.textContent);
+      });
```

Comment:
This will target every `mc-obfuscated` element in the document which isn't ideal, we want to limit it to only the ones in the component.

You could fix this by adding `:id="uid"` on the `div` element and then doing this instead of `getElementsByClassName`: ```document.querySelectorAll(`#${this.uid} .mc-obfuscated`)```